ssm/ssm2.py

```diff
@@ -551,7 +553,15 @@ def handle_connect(self):
         connect to each in turn until successful.
         """
         if self._protocol == Ssm2.AMS_MESSAGING:
-            log.debug('handle_connect called for AMS, doing nothing.')
+            log.info("Using AMS version %s", pkg_resources.get_distribution('argo_ams_library').version)
+
+            log.info("Establishing connection to %s", self._brokers[0])
```

Comment:
This function doesn't handle connections for AMS, so we shouldn't imply that a connection is about to be established here (and AMS works a bit differently anyway). Maybe just "Will connect to..." like the destination message you did below.

Reply:
Thanks, changes made.